### PR TITLE
Feature/picture

### PR DESCRIPTION
**Pull request by dunyakirkali**

### CODE REVIEW

**Review by felipeflorencio — APPROVED** [review]

**Review by felipeflorencio — CHANGES_REQUESTED** [review]
Have the version for the pod

File: Podfile [hunk 1]
```diff
@@ -5,6 +5,7 @@ target '100DaysSmiling' do
   use_frameworks!
   pod 'OAuthSwift', '~> 2.0.0'
   pod 'Swinject'
+  pod 'SwiftyCam'
```
Comment (felipeflorencio) [inline Podfile]: Have the version

File: 100DaysSmiling/Info.plist [hunk 2]
```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSMicrophoneUsageDescription</key>
```
Comment (felipeflorencio) [inline 100DaysSmiling/Info.plist:5]: It's mandatory to have the microphone if we are not using? Just about the privacy and that some people have some concerns about

**Review by felipeflorencio — APPROVED** [review]

File: 100DaysSmiling/Info.plist (hunk 2 above)
Comment (dunyakirkali) [reply 100DaysSmiling/Info.plist:5]: For now, we'll have to live with it

**Review by felipeflorencio — APPROVED** [review]

**Pull request merged**